tests/data/eatonups_eaton-connectups.json

```diff
@@ -0,0 +1,1221 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": null,
+                    "sysObjectID": ".1.3.6.1.4.1.534.1",
+                    "sysDescr": "ConnectUPS Web/SNMP Card V4.36",
+                    "sysContact": "<private>",
+                    "version": "FP: 0.13  INV: 1.29",
+                    "hardware": "PW9130 1000VA-R",
+                    "features": null,
+                    "os": "eatonups",
+                    "type": "power",
+                    "serial": null,
```

Comment:
No serial to be found while at it?

Reply:
An update with the serial number, not perfect but it is present ...
![image](https://user-images.githubusercontent.com/84004634/144119961-937bd325-e9d8-4768-8251-86a91c2c87ae.png)